pacemaker.spec.in

```diff
@@ -169,7 +170,7 @@ It will run scripts at initialization, when machines go up or down,
 when related resources fail and can be configured to periodically check
 resource health.
 
-Available rpmbuild rebuild options:
+Available rpmbuild rebuild options (not necessarily applicable/with effect):
```

Comment:
The upstream spec is for people building on their own, so there's no telling what else they're compiling. If they want to build CMAN on RHEL 7, they can, as unlikely as it may be.